packages/hardhat/contracts/SingleUriMetadata.sol

```diff
@@ -4,13 +4,15 @@ pragma solidity ^0.8.6;
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "./interfaces/IMetadata.sol";
 
+error METADATA_IS_IMMUTABLE();
+
 /** Metadata contract where all NFTs get same metadata */
 contract SingleUriMetadata is IMetadata, Ownable {
     event URIChanged(string indexed newURI);
     event MetadataFrozen();
 
     string public uri;
-    bool public metadataFrozen;
+    bool metadataFrozen;
```

Comment:
ditto re: public. should we add a getter if this is being made internal?

Reply:
done